data/strings/strings.txt

```diff
@@ -16382,6 +16382,143 @@
+    uk = Органічні мапи підтримують не всі категорії. Наприклад, не можна додати міста, залізничні станції, дороги, озера або будівлі. Будь ласка, додавайте такі об'єкти безпосередньо через osm.org (використовуйте кнопку "Редагувати" у верхній панелі) або використовуйте спеціалізований додаток для редагування OSM.
+    vi = Bản đồ hữu cơ không hỗ trợ tất cả các danh mục. Ví dụ. Không thể thêm thành phố, nhà ga, đường, hồ hoặc tòa nhà. Vui lòng thêm các tính năng đó trực tiếp qua osm.org (sử dụng nút Chỉnh sửa ở thanh trên cùng) hoặc sử dụng ứng dụng chỉnh sửa OSM chuyên dụng.
+    zh-Hans = 有机地图不支持所有类别。例如，无法添加城市、火车站、道路、湖泊或建筑物。请直接通过 osm.org（使用顶部栏的编辑按钮）或使用专门的 OSM 编辑应用程序添加此类功能。
+    zh-Hant = 有機地圖不支援所有類別。例如。無法新增城市、火車站、道路、湖泊或建築物。請直接透過 osm.org 新增此類功能（使用頂部欄中的「編輯」按鈕）或使用專門的 OSM 編輯應用程式。
+
+  [editor_category_unsuitable_text2]
+    tags = android
+    en = Alternatively, you can also use OSM Notes (osm.org/wiki/Notes) for reports to the OSM Community.
+    af = Alternatiewelik kan jy ook OSM Notes (osm.org/wiki/Notes) gebruik vir verslae aan die OSM-gemeenskap.
+    ar = وبدلاً من ذلك، يمكنك أيضًا استخدام OSM Notes (osm.org/wiki/Notes) لتقديم التقارير إلى مجتمع OSM.
+    az = Alternativ olaraq, OSM Birliyinə hesabatlar üçün OSM Qeydlərindən (osm.org/wiki/Notes) istifadə edə bilərsiniz.
+    be = Акрамя таго, вы таксама можаце выкарыстоўваць нататкі OSM (osm.org/wiki/Notes) для справаздач супольнасці OSM.
+    bg = Можете също така да използвате OSM Notes (osm.org/wiki/Notes) за доклади до общността на OSM.
+    ca = Alternativament, també podeu utilitzar OSM Notes (osm.org/wiki/Notes) per als informes a la comunitat OSM.
+    cs = Pro zprávy komunitě OSM můžete také použít službu Poznámky OSM (osm.org/wiki/Notes).
+    da = Alternativt kan du også bruge OSM Notes (osm.org/wiki/Notes) til rapporter til OSM-fællesskabet.
+    de = Alternativ kannst du auch OSM Notes (osm.org/wiki/Notes) für Hinweise an die OSM Community verwenden.
+    el = Εναλλακτικά, μπορείτε επίσης να χρησιμοποιήσετε το OSM Notes (osm.org/wiki/Notes) για αναφορές προς την κοινότητα του OSM.
+    es = También puedes utilizar Notas OSM (osm.org/wiki/Notes) para informar a la Comunidad OSM.
+    es-MX = Alternativamente, también puede utilizar Notas de OSM (osm.org/wiki/Notes) para informes a la Comunidad OSM.
+    et = Alternatiivina võite kasutada OSMi märkmeid (osm.org/wiki/Notes) ka OSMi kogukonnale esitatavate aruannete jaoks.
+    eu = Bestela, OSM Notes (osm.org/wiki/Notes) ere erabil dezakezu OSM Komunitateari txostenak egiteko.
+    fa = همچنین میu200cتوانید از یادداشتu200cهای OSM (osm.org/wiki/Notes) برای گزارشu200cهایی به انجمن OSM استفاده کنید.
+    fi = Vaihtoehtoisesti voit myös käyttää OSM Notesia (osm.org/wiki/Notes) OSM-yhteisölle tehtäviin raportteihin.
+    fr = Tu peux aussi utiliser OSM Notes (osm.org/wiki/Notes) pour les rapports destinés à la communauté OSM.
+    he = לחלופין, אתה יכול גם להשתמש ב-OSM Notes (osm.org/wiki/Notes) לדיווחים לקהילת OSM.
```

Comment:
Edit for `he`:
```suggestion
    he = לחלופין, ניתן להוסיף הערות למפה (osm.org/wiki/Notes) כדי לדווח לקהילת OSM.
```

Q: Does Organic Maps allow users to add notes to the map? If not I don't see the value of this string, or at least it needs to be clarified that OM doesn't support this either.